prisma/seed.ts

```diff
@@ -0,0 +1,59 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+async function main() {
+  await prisma.userMetadata.upsert({
+    where: { id: "00000000-0000-0000-0000-000000000000" },
+    update: {},
+    create: {
+      id: "00000000-0000-0000-0000-000000000000",
+      is_admin: true,
+      is_hr: false,
+      position: "Manager",
+    },
+  });
+
+  await prisma.userMetadata.upsert({
+    where: { id: "00000000-0000-0000-0000-000000000001" },
+    update: {},
+    create: {
+      id: "00000000-0000-0000-0000-000000000001",
+      is_admin: false,
+      is_hr: false,
+      position: "Employee",
+    },
+  });
+
+  await prisma.userMetadata.upsert({
+    where: { id: "00000000-0000-0000-0000-000000000002" },
+    update: {},
+    create: {
+      id: "00000000-0000-0000-0000-000000000002",
+      is_admin: false,
+      is_hr: false,
+      position: "Employee",
+    },
+  });
+
+  await prisma.userMetadata.upsert({
+    where: { id: "00000000-0000-0000-0000-000000000003" },
+    update: {},
+    create: {
+      id: "00000000-0000-0000-0000-000000000003",
+      is_admin: true,
```

Comment:
@MarcusCHill how should HR functionality differentiate from Administrative functionality? Should HR have administrative power?